moja-kuhinjica-frontend/.eslintrc.json

```diff
@@ -30,6 +30,7 @@
         "import/no-duplicates": "off",
         "no-useless-escape": "off",
         "prettier/prettier": "off",
-        "@typescript-eslint/no-unused-vars": "off"
+        "@typescript-eslint/no-unused-vars": "off",
+        "@typescript-eslint/restrict-plus-operands": "off"
```

Comment:
Why did you use this typescript rule in eslint?

Reply:
I turned it off while coding for easier building, now I use it again.